mcs/class/Mono.Posix/Mono.Unix.Native/Syscall.cs

```diff
@@ -727,6 +727,194 @@ public enum MremapFlags : ulong {
 		MREMAP_MAYMOVE = 0x1,
 	}
 
+	[Map]
+	[CLSCompliant (false)]
+	public enum SockType : int {
```

Comment:
See [google](https://www.google.com/search?q=%22using+system.net.sockets%22+%22using+mono.unix.native%22&prmd=ivns&filter=0) for some collisions.
In the mono repository itself there only one in [mcs/class/System.Windows.Forms/System.Windows.Forms/XplatUIX11.cs](https://github.com/mono/mono/blob/a8cb543870c1177185e212e27e50ccdee5f4d60b/mcs/class/System.Windows.Forms/System.Windows.Forms/XplatUIX11.cs).
Basically it's people using networking plus some functionality from Mono.Unix.Native.